composer.json

```diff
@@ -9,12 +9,12 @@
 		"ext-simplexml": "*"
 	},
 	"require-dev": {
-	    "dealerdirect/phpcodesniffer-composer-installer": "0.7.0",
-	    "phpcompatibility/phpcompatibility-wp": "2.1.0",
-	    "squizlabs/php_codesniffer": "3.5.3",
-	    "wp-coding-standards/wpcs": "2.2.0",
 		"wp-phpunit/wp-phpunit": "^5.8",
-		"yoast/phpunit-polyfills": "^1.0"
+		"yoast/phpunit-polyfills": "^1.0",
+		"dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
+		"phpcompatibility/phpcompatibility-wp": "^2.1",
+		"squizlabs/php_codesniffer": "^3.7",
+		"wp-coding-standards/wpcs": "^2.3"
```

Comment:
Part 1 of the fix.

Updating these packages fixed the initial `phpcs` error we saw, like:

```sh
Deprecated: strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated in wp-content/wpps-scripts/vendor/squizlabs/php_codesniffer/src/Util/Common.php on line 42
```